src/questions/data_types.yaml

```diff
@@ -1,37 +1,1106 @@
+  hint: address holds a 20 byte value.
+  explanation: address payable is same as address but with aditional members send and transfer.
+  answerKeys:
+    - A
+    - B
+    - D
+  sub_topics:
+    - "value-types"
+  difficulty_level: Medium
+  choices:
+    - content: Address payable has a larger size compared to address.
+      key: A
+    - content: Implicit type conversion from address payable to address are not allowed.
+      key: B
+    - content: If you need a variable of type address and plan to send Ether to it, then declare its type as address payable.
+      key: C
+    - content: Explicit conversion of data type byte 32 to address is allowed.
+      key: D
+
+- uuid: d5773538-949b-4909-b91e-5f035808b8bc
+  type: SingleChoice
+  content: Which of the following is not a valid member of type address?
+  hint: noHint
+  explanation: keccak256 is a globally available function in solidity used to calculate the keccak256 hash of an input . balance , codehash and transfer are methods of the address type.
+  answerKeys:
+    - B
+  sub_topics:
+    - "value-types"
+  difficulty_level: Low
+  choices:
+    - content: balance
+      key: A
+    - content: keccak256
+      key: B
+    - content: codehash
+      key: C
+    - content: transfer
+      key: D
+
+- uuid: b6a45240-74b1-4243-b3b1-9c42c154be7e
+  type: MultipleChoice
+  content: Which of the following are true regarding constant and immutable variables in solidity?
+  hint: these variables cannot be modified after the contract has been constructed.
+  explanation: For constant variables, the value has to be a constant at compile time and it has to be assigned where the variable is declared whearas immutables are a little less restrictive and can be assigned an arbitrary value in the constructor of the contract or at the point of their declaration.        
+  answerKeys:
+    - A
+    - C
+  sub_topics:
+    - "value-types"
+  difficulty_level: Medium
+  choices:
+    - content: For constant variables, the value has to be fixed at compile-time. For immutable, value can be assigned at construction time.
+      key: A
+    - content: Compared to regular state variables, the gas costs of constant & immutable variables are much higher.
+      key: B
+    - content: Variables declared as immutable are a bit less restricted than those declared as constant.
+      key: C
+    - content: Immutable variables can be assigned more than once.
+      key: D
+
+- uuid: b0c8e946-2740-499a-9863-3173f1dc8a52
+  type: SingleChoice
+  content: in solidity 0xdCad3a6d3569DF655070DEd06cb7A1b2Ccd1D3AF is an example of-
+  hint: noHint
+  explanation: addresses holds 20 byte hexadecimal literals.
+  answerKeys:
+    - A
+  sub_topics:
+    - "value-types"
+  difficulty_level: Low
+  choices:
+    - content: address
+      key: A
+    - content: uint
+      key: B
+    - content: string
+      key: C
+    - content: array
+      key: D
+
+- uuid: 944228da-440e-43d3-b0b8-a9a3751aa5b9
```

Comment:
If you add the question and include `<code>` or ```<multi line code>``` it is formatted nicely by markdown

```
- uuid: 944228da-440e-43d3-b0b8-a9a3751aa5b9
  type: SingleChoice
  content: which of the following is not a valid array declaration?
  hint: noHint
  explanation: type arrayName [ arraySize ]; declares an array of fixed size in Solidity,
               type[] arrayName; declares an array of dynamic size in solidity.
  answerKeys:
    - D
  sub_topics:
    - "reference-types"
  difficulty_level: Low
  choices:
    - content: "`uint arr[10];`"
      key: A
    - content: "`int []arr;`"
      key: B
    - content: "`uint arr[] = [1,2,3];`"
      key: C
    - content: "`int [3]arr;`"
      key: D
```

We then have to wrap with double quotes to support yaml format. So the inside string is in markdown format and to support that in yaml we have to wrap it with double quotes